content/en/docs/concepts/configuration/secret.md

```diff
@@ -143,7 +143,7 @@ method creates a new `Secret` object with the edited data.
 
 Depending on how you created the Secret, as well as how the Secret is used in
 your Pods, updates to existing `Secret` objects are propagated automatically to
-Pods that use the data. For more information, refer to [Mounted Secrets are updated automatically](#mounted-secrets-are-updated-automatically).
+Pods that use the data. For more information, refer to [Mounted Secrets are updated automatically](#using-secrets-as-files-from-a-pod).
```

Comment:
```suggestion
Pods that use the data. For more information, refer to [Using Secrets as files from a Pod](#using-secrets-as-files-from-a-pod) section.
```

IMO the referenced text should be changed. As a user, I'd like to be navigated to a link that actually contains the required heading else it'll look like other issue or the user might think that link is not pointing to correct place because the reference text don't match with actual heading.

Reply:
@NitishKumar06 I did change the reference text before this commit , you said not to change so I changed it back to the previous one. Now should I change it back ?

Reply:
@NitishKumar06 please watch out for making suggestions that replace valid grammar with something unidiomatic.
I / we appreciate the intent, but contributors might be misled by a review into making an unhelpful “fix”.